tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # Download Bazel
</pre><p>Remove the bazel downloading part?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Removed